src/thermo/PDSS_ConstVol.cpp

```diff
@@ -78,12 +78,12 @@ void PDSS_ConstVol::setState_TP(doublereal temp, doublereal pres)
     setPressure(pres);
 }
 
-void PDSS_ConstVol::setState_TR(doublereal temp, doublereal rho)
+void PDSS_ConstVol::setState_TD(double temp, double rho)
```

Comment:
Should we consider remaining these parameters at the same time? Density instead of rho?

Reply:
I don't think this to be necessary